go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/go-mysql-org/go-mysql
 
-go 1.18
+go 1.22
```

Comment:
So 1.22 and 1.23 are in supported status now and 1.21 is not anymore. 

https://go.dev/doc/devel/release